maven/lib/dependabot/maven/utils/auth_headers_finder.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Dependabot
+  module Maven
+    module Utils
+      class AuthHeadersFinder
+        def initialize(credentials)
+          @credentials = credentials
+        end
+
+        def auth_headers(maven_repo_url)
```

Comment:
This feels like an absolute nit-pick, but since we don't expect to pass different `maven_repo_url`s to this method, it seems slightly more clear to pass the url in the constructor and calling `auth_headers` without arguments. WDYT?

Reply:
As mentioned, there will be cases where multiple calls are made, with different URLs, tests did not catch this.

Reply:
Ah I missed that 👍